auto/backup/uploader.go

```diff
@@ -173,6 +173,7 @@ func (u *Uploader) upload(ctx context.Context) error {
 		stats.Get(lastUploadBytes).(*expvar.Int).Set(cr.Count())
 		u.lastUploadTime = time.Now()
 		u.lastUploadDuration = time.Since(startTime)
+		u.logger.Printf("completed auto upload to %s in %s", u.storageClient, u.lastUploadDuration)
```

Comment:
I'm concerned this log message is too noisy since it's periodic, and doesn't add much value. I try to avoid periodic repeating log messages if possible. The "last upload duration" is actually available from /status I believe (and if it's not, it should be). Or added to the /debug/vars output.

I'm open to convincing, but this data is available already I think.

Reply:
Ok, let me take a stab at convincing you:

Logs are used to signal important events taken by software. If I'm exporting, compressing, and shipping a copy of the database out to cloud storage, this for me qualifies as pretty important. While it's true it can be seen in the metrics (although getting those metrics into a system like Prometheus is currently challenging), logging this kind of event is very useful diagnostically.  For example if I see a spike in iowait% or cpu I might be wondering why, and the log line quickly, conveniently, and fully explains it.

As a operator, I would never complain about this kind of log event. I work with software that logs all kinds of useless crap not even the devs care about when I press them (sometimes even at WARN level!), but one log line every few hours or even minutes to signal an event that can have significant bearing on system resources is definitely not in that camp. :)

Reply:
I hasten to add it's terribly uncharacteristic of me to argue in favor of *more* logging than less, but suffice it to say if you remain unconvinced I can drop it from the PR.

Reply:
All good, let's leave it so.